ascify art/Ascify-Art.py

```diff
@@ -119,137 +119,140 @@ def getChar(inputInt):
         if len(dir_)>=30:
-            entry_location.configure(text=dir_[:25]+"..."+dir_[-5:])
+            entry_location.configure(text=f"{dir_[:25]}...{dir_[-5:]}")
         else:
             entry_location.configure(text=dir_)
-            
-        var.set(os.path.basename(file)[:-4]+"_Ascified")
-        
+
+        var.set(f"{os.path.basename(file)[:-4]}_Ascified")
+
         try:
             root.unbind("<KeyRelease-Left>")
             root.unbind("<KeyRelease-Right>")
             RightClickMenu.delete("Next Frame")
             RightClickMenu.delete("Previous Frame")
         except:
             None
-    else:
-        if previous!="":
-            file = previous
-            
+    elif previous!="":
+        file = previous
+
 def frame_next():
     # load next frame of sequence
     global frame_no, file
     if frame_no<len(allitems)-1:
         file = allitems[frame_no+1]
         frame_no+=1
         operation()
-    
+
 def frame_previous():
     # load previous frame of sequence
     global frame_no, file
     if frame_no>0:
         file = allitems[frame_no-1]
         frame_no-=1
         operation()
-  
+
 def open_sequence():
     # opening a sequence of images (folder)
-    
+
     global dir_, file, img, image, allitems, sequence
-    dir_img = filedialog.askdirectory()
-    if dir_img:
-        
+    if dir_img := filedialog.askdirectory():
         allitems = glob.glob(dir_img+'\*.png')
         allitems.extend(glob.glob(dir_img+'\*.jpeg'))
         allitems.extend(glob.glob(dir_img+'\*.jpg'))
         allitems.extend(glob.glob(dir_img+'\*.bmp'))
         allitems.extend(glob.glob(dir_img+'\*.webp'))
-        
+
         if len(allitems) == 0:
             tk.messagebox.showinfo("Oops!", "No valid image files present in this folder!")
             return
-        
+
         sequence = True
         if len(dir_img)>=50:
-            open_button.configure(text=dir_img[:40]+"..."+dir_img[-3:])
+            open_button.configure(text=f"{dir_img[:40]}...{dir_img[-3:]}")
         else:
             open_button.configure(text=dir_img)
-            
+
         dir_ = os.path.dirname(dir_img)
         if len(dir_)>=30:
-            entry_location.configure(text=dir_[:25]+"..."+dir_[-5:])
+            entry_location.configure(text=f"{dir_[:25]}...{dir_[-5:]}")
         else:
             entry_location.configure(text=dir_)
-            
+
         try:
             RightClickMenu.delete("Next Frame")
             RightClickMenu.delete("Previous Frame")
         except:
             None
-            
-        var.set(os.path.basename(dir_img)+"_Ascified")
+
+        var.set(f"{os.path.basename(dir_img)}_Ascified")
         RightClickMenu.add_command(label="Next Frame", command=lambda: frame_next())
         RightClickMenu.add_command(label="Previous Frame", command=lambda: frame_previous())
-        
+
         file = allitems[0]
-        img = Image.open(file)   
+        img = Image.open(file)

```

Comment:
Function `open_sequence` refactored with the following changes:

- Use named expression to simplify assignment and conditional
- Use f-string instead of string concatenation [x5]